lib/ui/components/captured-requests.js

```diff
@@ -102,11 +100,15 @@ export class CapturedRequests extends React.Component {
               <ActiveIndicator active={ this._isAlreadyMocked(request) }
                                onToggle={ () => {} }/>
               <MethodLabel method={ request.method }/>
-              { this._requestDetailsFor(request) }
+              <Url url={ request.url }/>
             </div>
 
-            { this._actionButtonFor(request) }
+            <div className="buttons">
+              { this._actionButtonFor(request) }
+              <button className="btn btn-mock" onClick={ this._mockRequest.bind(this, request) }>Mock</button>
```

Comment:
Can you provide a link to the comment or line?

Currently I see this change as a broken UI. You can see the mock and edit buttons at the same time.
Clicking the mock button more than once causes duplicate requests in the mocked requests section which isn't the intended behaviour. 

<img width="775" alt="screenshot 2016-01-27 00 06 21" src="https://cloud.githubusercontent.com/assets/1665570/12604487/073def18-c48a-11e5-853e-dde4c703fc1d.png">